docs/layers.md

```diff
@@ -853,6 +861,7 @@ To resolve inconsistency in data tagging in OpenStreetMap we normalize several o
 * `consulting`
 * `convenience`
 * `copyshop` - A shop offering photocopying and printing services.
+* `cosmetics` - A shop selling cosmetics.
```

Comment:
Since we also say chemist sells "cosmetics" let's say for this one:
`A specialty shop selling cosmetics.`

Reply:
Good point, thanks. Fixed in fe1cbf0.